pkg/ds/daemon_set.go

```diff
@@ -293,6 +302,11 @@ func (ds *daemonSet) addPods() error {
 			return util.Errorf("Error scheduling node: %v", err)
```

Comment:
now that the scheduling is being done in replication, you should rename this error message to "labeling" or someting along those lines (and rename `schedule()` to label)